spec/emerald/rubify_spec.rb

```diff
@@ -0,0 +1,131 @@
+require "spec_helper"
+require "emerald/rubify"
+require "emerald/atom"
+require "emerald/string"
+require "emerald/number"
+require "emerald/list"
+
+RSpec.describe Emerald::Rubify do
+
+  it "compiles an empty source" do
+    compiled_code = Emerald::Rubify.new([]).rubify
+    expect(compiled_code).to eq('')
+  end
+
+  describe "code generation from atoms" do
+    it "generates code from a single atom" do
+      compiled_code = Emerald::Rubify.new([Emerald::Atom.new("hello-world")]).rubify
+      expect(compiled_code).to eq('hello-world')
+    end
+
+    it "generates code from two atoms" do
+      compiled_code = Emerald::Rubify.new([Emerald::Atom.new("hello"),Emerald::Atom.new("world")]).rubify
+      expect(compiled_code).to eq('hello world')
```

Comment:
Ah right, I think I see what you mean - because everything in a lisp should be within a list, it doesn't make sense to write tests in which there are atoms/numbers/strings not included within a list? I think I'd taken the parser tests a bit too literally and started testing here by implementing the inverse of those tests. Makes sense 👍